src/v2/Apps/Settings/Routes/EditSettings/Components/SettingsEditSettingsTwoFactor/TwoFactorAuthentication/Components/SmsSecondFactor/index.tsx

```diff
@@ -129,66 +137,81 @@ export const SmsSecondFactor: React.FC<SmsSecondFactorProps> = ({
     })
   }
 
+  const show2FAWarning = /@artsymail.com$/.test(me.email!)
```

Comment:
**question**: do you think it would be a good idea to take this out into a separate reusable function and cover it tests?

Something like this
```javascript
const isArtsyEmail = (email: string) => {
  return /@artsymail.com$/.test(email)
}
```

Reply:
sure, will do!

Reply:
thank you!